sample_apps/simple_app_under_keystone/bare_app.cc

```diff
@@ -72,22 +71,22 @@ bool keystone_test(const int cert_size, byte *cert) {
     if (!keystone_Attest(size_what_to_say, what_to_say,
                          &size_attestation, attestation)) {
         printf("keystone_Attest() fails\n");
-        return false;
+        success = false;
     }
     if (!keystone_Verify(size_what_to_say, what_to_say,
                          size_attestation, attestation, &size_measurement, measurement)) {
         printf("keystone_Verify() fails\n");
-        return false;
+        success = false;
     }
     printf("Measurement: ");
     for (int i = 0; i < size_measurement; i++) {
         printf("%x", measurement[i]);
     }
     printf("\n");
 
-    return true;
+    return success;
 }
 
 int main(int argc, char** argv) {
```

Comment:
nit: Let's print out the final result (currently it prints nothing if everything passes)

Reply:
resolved